packages/file_selector/file_selector_platform_interface/lib/src/method_channel/method_channel_file_selector.dart

```diff
@@ -79,18 +76,32 @@ class MethodChannelFileSelector extends FileSelectorPlatform {
     );
   }
 
-  /// Gets a directory path from a dialog
   @override
   Future<String?> getDirectoryPath({
     String? initialDirectory,
     String? confirmButtonText,
   }) async {
-    return _channel.invokeMethod<String>(
+    final List<String>? pathList = await _channel.invokeListMethod<String>(
```

Comment:
The default implementation's channel details are effectively public API, so this is a breaking change. For the default implementation it needs to be a new method with different semantics.

In the implementation packages, by contrast, the channel details are private so we can change them whenever we want to; that's part of why we don't use the shared method channels in 1P implementations any more.

Reply:
So we should leave this as it used to be?